man/systemd.network.xml

```diff
@@ -3089,6 +3089,14 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
         for details. Defaults to <literal>medium</literal>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>HopLimit=</varname></term>
+        <listitem>
+          <para>Configures IPv6 router hop limit. Takes an integer in the range 1…255. See also
```

Comment:
`IPv6 router hop limit` sounds spurious. How about to simply `hop limit` or `IPv6 hop limit` ?

Also, the parser also accepts 0.

Reply:
Yes updated.